README.md

```diff
@@ -44,4 +44,9 @@ Once your adapter is built out a few options.
 
 ## Code of Conduct
 
-Everyone interacting in the dbt project's codebases, issue trackers, chat rooms, and mailing lists is expected to follow the [dbt Code of Conduct](https://community.getdbt.com/code-of-conduct).
\ No newline at end of file
+Everyone interacting in the dbt project's codebases, issue trackers, chat rooms, and mailing lists is expected to follow the [dbt Code of Conduct](https://community.getdbt.com/code-of-conduct).
+
+
+<!-- "directory_name": "{{cookiecutter.adapter_name.lower().replace(' ','')}}",
```

Comment:
What are these bottom lines here for?

Reply:
sorry those were hold overs from a different branch that should of gone away, removed